controllers/validator.go

```diff
@@ -18,12 +18,18 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 		return false, errors.New("DPA CR Velero configuration cannot be nil")
 	}
 
-	if len(dpa.Spec.BackupLocations) == 0 && !dpa.Spec.Configuration.Velero.NoDefaultBackupLocation {
-		return false, errors.New("no backupstoragelocations configured, ensure a backupstoragelocation has been configured or use the noDefaultLocationBackupLocation flag")
+	if dpa.Spec.Configuration.Velero.NoDefaultBackupLocation {
+		if len(dpa.Spec.BackupLocations) != 0 {
```

Comment:
Not sure this is true, we really are just saying if we have any BSLs defined in the DPA spec that none of them have `default: true` right?

Reply:
after digging, `velero install` takes the same approach https://github.com/vmware-tanzu/velero/blob/77be862b9c8d4d0b9187a8239059e6ed9b54cd1b/pkg/cmd/cli/install/install.go#L307-L309

so its fine

Reply:
We only use install lib for the deployment, so we really can choose our own behavior here. Awaiting merge for more input

Reply:
@shubham-pampattiwar this flag is only used in [func AllResources](https://github.com/vmware-tanzu/velero/blob/cb11bf1924273428890b4f75e012c6e18cf8fed7/pkg/install/resources.go#L251) which we skipped entirely from using.